SH4RE/SH4RE/Components/Stars.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Stars.swift
+//  SH4RE
+//
+//  Created by Hannah Brooks on 2022-11-30.
+//
+
+import SwiftUI
+
+struct StarsView: View {
+    private var numberOfStars: Int
+    private var hasHalfStar: Bool
+    
+    init(numberOfStars: Int, hasHalfStar: Bool) {
+        self.numberOfStars = numberOfStars
+        self.hasHalfStar = hasHalfStar
+    }
+    
+    var body: some View {
+        ForEach(0 ..< self.numberOfStars, id: \.self) { idx in
```

Comment:
not sure that self is needed here since there are no other numberOfStars in this scope, only in the init. Also, is the id needed in this ForEach?

Reply:
i think i just used self out of habit I can change. and yes id is needed. throws an error otherwise